astropy/cosmology/units.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""Cosmological units and equivalencies.
+"""  # (needed for unit summary)
+
+__all__ = ["littleh", "with_H0"]
+
+from astropy.units.utils import generate_unit_summary as _generate_unit_summary
+
+###############################################################################
+# Cosmological Units
+# isort: split
+
+from astropy.units.astrophys import littleh
+
+###############################################################################
+# Equivalencies
+# isort: split
+
+from astropy.units.equivalencies import with_H0
+
+# =============================================================================
+# DOCSTRING
+
+# This generates a docstring for this module that describes all of the
+# standard units defined here.
+if __doc__ is not None:
+    __doc__ += _generate_unit_summary(globals())
```

Comment:
Would it be an idea to define `enable()` (as in `imperial.py`, etc.)?

Reply:
Let me take a look! that seems a good idea.

Reply:
My second thought was perhaps not (see below), but do have your own look, since clearly I am wavering!